### PR TITLE
docs: constraints should work for all managers

### PR DESCRIPTION
**Pull request by viceice**
## Changes
Remove the outdated `contraints` supported manager list.
`constraints` should work for all managers.

## Context
- #21588

## Documentation (please check one with an [x])

- [x] I have updated the documentation, or
- [ ] No documentation update is required

## How I've tested my work (please select one)

I have verified these changes via:

- [x] Code inspection only, or
- [ ] Newly added/modified unit tests, or
- [ ] No unit tests but ran on a real repository, or
- [ ] Both unit tests + ran on a real repository

### CODE REVIEW

**Review by secustor — APPROVED** [review]

**Review by HonkingGoose — COMMENTED** [review]
I don't know how to even check this PR. I'll let the other maintainers review. 😉

**Pull request merged**
